guides/common/modules/proc_configuring-a-kickstart-template-to-distribute-ssh-keys-during-provisioning.adoc

```diff
@@ -1,14 +1,14 @@
-[id="configuring-a-kickstart-template-to-distribute-ssh-keys-during-provisioning_{context}"]
-= Configuring a Kickstart Template to Distribute SSH Keys during Provisioning
+[id="configuring-a-kickstart-template-to-distribute-script-keys-during-provisioning_{context}"]
+= Configuring a Kickstart Template to Distribute Script Keys during Provisioning
```

Comment:
Similarly, the keys are specific to the transport mechanism (SSH) so it should stay the way it was. The same goes for all the other references to keys in this doc